bin/ff_map

```diff
@@ -81,8 +81,27 @@ def main():
         treated_atoms = np.array([])
 
     print("INFO - Loading mapping files")
-    with open(args.map_file, "r") as _file:
-        lines = _file.readlines()
+    #determine if we have a single .mapping file or a directory of .map files
+    map_path = pathlib.Path(args.map_file)
+    if map_path.is_file() == True:
+        with open(args.map_file, "r") as _file:
+            lines = _file.readlines()
+    elif map_path.is_dir() == True:
+        l = []
+        for i in map_path.glob('*.map'):
+            with open(i, "r") as _file:
+                l.append(_file.readlines())
+                l.append('\n')
+        if len(l) > 0:
+            lines = [item for sublist in l for item in sublist]
+            with open("full_output.mapping", "w") as opfile:
+                opfile.writelines(lines)
```

Comment:
I don't get this part fully? Why re we now writing lines?

Reply:
left over from testing oops